{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:ibm:cognos_tm1:10.1.0:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:ibm:cognos_tm1:10.1.0.1:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:ibm:cognos_tm1:10.1.1:*:*:*:*:*:*:*",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2013-0484"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "product": {
              "product_data": [
                {
                  "product_name": "cognos_tm1",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "=",
                        "version_value": "10.1.0"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "10.1.0.1"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "10.1.1"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "ibm"
          }
        ]
      }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "The server process in IBM Cognos TM1 10.1.x before 10.1.1 FP1 allows remote attackers to cause a denial of service (daemon crash) via an undocumented API call that triggers the transmission of unexpected data."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "NVD-CWE-noinfo"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "http://www-01.ibm.com/support/docview.wss?uid=swg21637655",
          "refsource": "CONFIRM",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "http://www-01.ibm.com/support/docview.wss?uid=swg21637655"
        },
        {
          "name": "tm1-undocumented-api(81612)",
          "refsource": "XF",
          "tags": [],
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/81612"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "accessComplexity": "MEDIUM",
        "accessVector": "NETWORK",
        "authentication": "NONE",
        "availabilityImpact": "PARTIAL",
        "baseScore": 4.3,
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "vectorString": "AV:N/AC:M/Au:N/C:N/I:N/A:P",
        "version": "2.0"
      },
      "exploitabilityScore": 8.6,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainOtherPrivilege": false,
      "obtainUserPrivilege": false,
      "severity": "MEDIUM",
      "userInteractionRequired": false
    }
  },
  "lastModifiedDate": "2017-08-29T01:33Z",
  "publishedDate": "2013-06-19T13:39Z"
}